java/opendcs/src/main/java/decodes/launcher/LauncherFrame.java

```diff
@@ -1193,11 +1193,24 @@ private void doInitDecodes() throws DecodesException
         try
         {
             // Load the DCS Tool Configuration
-            DecodesSettings settings = DecodesSettings.fromProfile(launchProfile);
+            final DecodesSettings settings = DecodesSettings.fromProfile(launchProfile);
             databases = DatabaseService.getDatabaseFor(null, settings);
+            databases.getLegacyDatabase(Database.class).ifPresent(db ->
+            {
+                try
+                {
+                    db.initMinimal(settings);
+                    Database.setDb(db);
+                }
+                catch (DecodesException ex)
+                {
+                    throw new RuntimeException(ex);
+                }
+            });
+
 
         }
-        catch (Exception ex)
+        catch (Throwable ex)
```

Comment:
Should catch `RuntimException`, not `Throwable`. Unless there's a specific `Error` subclass you're targeting (scary), which should get a comment explainer

Reply:
fair point, I was mostly being lazy but with `catch( Type 1 | Type 2 ex)` syntax there's not really any point in being that sloppy.